api/v1alpha1/nicclusterpolicy_types.go

```diff
@@ -67,8 +67,19 @@ type ConfigMapNameReference struct {
 	Name string `json:"name,omitempty"`
 }
 
+// InitContainerSpec contains configuration for the init container of the OFED driver
+type InitContainerSpec struct {
+	// Enable indicates if init container deployment is required
+	// +optional
+	// +kubebuilder:default:=false
+	Enable    bool `json:"enable,omitempty"`
```

Comment:
You can't use safeLoad feature without the init container. The idea is that now init container is required only for this feature, but then it will be used for other features as well

Reply:
Ack.
Consider add a check in the admission controller that if SafeLoad is enabled, InitContainer is also enabled.